src/scss/_global/_structure.scss

```diff
@@ -3,6 +3,8 @@ body {
   display: flex;
   flex-direction: column;
   justify-content: flex-start;
+  overflow-x: hidden;
```

Comment:
I wouldn't do this, just because we need to know if something we're doing is breaking the layout.